framework/sessions/__init__.py

```diff
@@ -150,6 +151,8 @@ def before_request():
             session.data['auth_user_username'] = user.username
             session.data['auth_user_id'] = user._primary_key
             session.data['auth_user_fullname'] = user.fullname
+            user.date_last_login = datetime.utcnow()
+            user.save()
```

Comment:
@sloria, going to do a functional test on this, pushing so Travis can do his thing.